### ISSUE TITLE
Segmentation fault. Core dumped.

### ISSUE THREAD

**Issue by ZeroMaxinumXZ** [keras-team/keras#12574]
I'm having issues with Keras. Basically, it gives me the following error "Segmentation fault (core dumped)" when I try to fit a model with a conv2d layer. 

My code works on the CPU. It also works without any conv2d layers (even though it's ineffective for my use case). I've got cuda, cudnn, and tensorflow installed. I've tried reinstalling keras and tensorflow.

Code sample:
`
import numpy as np
import keras

model = keras.models.Sequential() #Sequential model type.
model.add(keras.layers.Conv2D(filters=1, kernel_size=(3,3), strides = 1, activation="sigmoid")) #Convolutional layer.
model.add(keras.layers.Flatten()) #Flatten layer.
model.add(keras.layers.Dense(4)) #Dense layer of 4 units.
model.compile(loss='mean_squared_error', optimizer='adam') #compile model.
y = np.random.rand(1,4) #Random expected output
x = np.random.rand(1, 38, 21, 1) # Random input.
model.fit(x, y) #And fit...
`

Error:
`
Using TensorFlow backend.
Epoch 1/1
2019-03-29 03:14:25.259882: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-03-29 03:14:25.573157: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:964] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2019-03-29 03:14:25.574989: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 0 with properties:
name: GeForce RTX 2060 major: 7 minor: 5 memoryClockRate(GHz): 1.83
pciBusID: 0000:01:00.0
totalMemory: 5.73GiB freeMemory: 5.49GiB
2019-03-29 03:14:25.575033: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1511] Adding visible gpu devices: 0
2019-03-29 03:14:31.770903: I tensorflow/core/common_runtime/gpu/gpu_device.cc:982] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-03-29 03:14:31.770974: I tensorflow/core/common_runtime/gpu/gpu_device.cc:988]      0
2019-03-29 03:14:31.770992: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 0:   N
2019-03-29 03:14:31.772113: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 5235 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2060, pci bus id: 0000:01:00.0, compute capability: 7.5)
Segmentation fault (core dumped)
`

**Comment by jvishnuvardhan** [comment]
@ZeroMaxinumXZ What is the version of TF and Keras? Thanks!

**Comment by ZeroMaxinumXZ** [comment]
Keras version: 'v2.1.6-tf'

Tensorflow-GPU version: 'v1.12'

Python version: 'v3.5.2'

CUDA version: 'v9.0.176'

CUDNN version: 'v7.2.1.38-1+cuda9.0

**Comment by ZeroMaxinumXZ** [comment]
Just had to downgrade cuDNN to version v7.1.2... Issue solved.

**Comment by barzan-hayati** [comment]
@ZeroMaxinumXZ   I had installed this version on Ubuntu 18.04 and I receive this message 
`Segmentation fault (core dumped)`
for any model that i want to train it, even for training MNIST without any changes in it model. 
`cudatoolkit               9.2                           0    anaconda`
`cudnn                     7.2.1                 cuda9.2_0    anaconda`
`keras                     2.2.4                    pypi_0    pypi`
`keras-applications        1.0.8                      py_0    anaconda`
`keras-preprocessing       1.1.0                      py_1    anaconda`
`tensorboard               1.12.2           py36he6710b0_0    anaconda`
`tensorflow                1.12.0          gpu_py36he74679b_0    anaconda`
`tensorflow-base           1.12.0          gpu_py36had579c0_0    anaconda`
`tensorflow-gpu            1.12.0               h0d30ee6_0    anaconda`

Are you sure that I must downgrade `CUDNN` to `7.1.2`

**Comment by jvishnuvardhan** [comment]
@barzan-hayati try to downgrade cuda to 9.0 from 9.2 and test it. Thanks!

**Comment by andreyvgritsenko** [comment]
I also have the same issue (Segmentation fault (core dumped))
`# Name                    Version                   Build  Channel
cudatoolkit               9.0                  h13b8566_0  
cudnn                     7.1.2                 cuda9.0_0  
keras-applications        1.0.6                    py27_0  
keras-base                2.2.4                    py27_0  
keras-gpu                 2.2.4                         0  
keras-preprocessing       1.0.5                    py27_0  
tensorboard               1.10.0           py27hf484d3e_0  
tensorflow                1.10.0          gpu_py27h6f941b3_0  
tensorflow-base           1.10.0          gpu_py27h6ecc378_0  
tensorflow-gpu            1.10.0               hf154084_0  `

Any suggestions?
